VideoLocker/res/layout/fragment_course_combined_info.xml

```diff
@@ -171,19 +172,9 @@
                         tools:targetApi="17" />
 
                     <Switch
+                        style="@style/edX.Widget.Switch"
```

Comment:
If we're using this style as the app default, then this can be set globally in the theme via the `android:switchStyle` attribute, instead of applying it individually to each widget.